numpy/random/generator.pyx

```diff
@@ -3361,6 +3361,19 @@ cdef class Generator:
         tol : float, optional
             Tolerance when checking the singular values in covariance matrix.
             cov is cast to double before the check.
+        method : { 'svd', 'eigh', 'cholesky'}, optional
+            The cov input is used to compute a factor matrix A such that
+            A.dot(transpose(A)) = cov. This argument is used to select the method
```

Comment:
I find the below syntax to be clearer (closer to the maths than method & function calls)
```suggestion
            ``A @ A.T = cov``. This argument is used to select the method
```